crypto/stark-verifier-ethereum/contracts/primefield.sol

```diff
@@ -0,0 +1,71 @@
+pragma solidity ^0.6.4;
+
+contract PrimeField {
+    uint256 constant internal MODULUS = 0x800000000000011000000000000000000000000000000000000000000000001;
+    uint256 constant internal MODULUS_MASK = 0x0fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff;
+    uint256 constant internal MONTGOMERY_R = 0x7fffffffffffdf0ffffffffffffffffffffffffffffffffffffffffffffffe1;
+    uint256 constant internal MONTGOMERY_R_INV = 0x40000000000001100000000000012100000000000000000000000000000000;
+    uint256 constant internal GENERATOR = 3;
+    uint256 constant internal ONE = 1;
+
+    function from_montgomery(uint256 value) internal pure returns (uint256 result) {
```

Comment:
Inconsistent style: sometimes the return argument is named, sometimes (as below) it is not.